test/shared/src/main/scala/zio/test/TestRunner.scala

```diff
@@ -29,12 +29,14 @@ import zio.internal.Platform
 final case class TestRunner[R <: Has[_], E](
   executor: TestExecutor[R, E],
   platform: Platform = Platform.makeDefault().withReportFailure(_ => ()),
-  reporter: TestReporter[E] = DefaultTestReporter(TestAnnotationRenderer.default),
-  bootstrap: ZLayer.NoDeps[Nothing, TestLogger with Clock] = ((Console.live >>> TestLogger.fromConsole) ++ Clock.live)
+  renderer: TestRenderer[E] = DefaultTestRenderer(),
+  bootstrap: ZLayer.NoDeps[Nothing, TestLogger with Clock] = (Console.live >>> TestLogger.fromConsole) ++ Clock.live
 ) { self =>
 
   lazy val runtime = Runtime((), platform)
 
+  lazy val reporter = DefaultTestReporter(renderer, TestAnnotationRenderer.default)
```

Comment:
The reporter is now just passes rendered results to the logger provided by `TestLogger`. If someone wants to change it, I suppose this can be done by providing a different bootstrap layer.